collections/St_Johns_College/SW_V1_St_Johns_College_MS_006.xml

```diff
@@ -0,0 +1,233 @@
+                              Sch<supplied>row</supplied>sbury</persName></colophon>
+                     <author key="person_22158282">John Lydgate</author>
+                     <title key="work_2732">The Troy Book</title>
+                     <note><bibl>(IMEV 2516)</bibl>, <bibl>ed. Thomas Bergen, EETS ES 97, 103, 106
+                           (1906–10)</bibl> , with the envoy and ‘Verba translatoris’ (the actual
+                        explicit). Bergen describes our MS at <bibl>EETS ES 126 (1935) 36–8.</bibl>
+                        <locus from="134rb" to="134v">Fol. 134<hi rend="superscript"
+                           >rb-v</hi></locus> originally blank. This copy lacks Prol.210–1.49 on a
+                        lost leaf following <locus from="1r" to="1v">fol. 1</locus> (the original
+                           2<hi rend="superscript">o</hi> folio should have been a version of 1.50
+                        ‘Fordymmed eke the lettris aureat’). The scribe has left blanks marked with
+                        crosses for missing lines; cf. <locus from="43ra" to="43ra">fol. 43<hi
+                              rend="superscript">ra</hi></locus>, with the informal <term
+                           type="script">anglicana</term> note ‘caret versus’ <date
+                           calendar="Gregorian" notBefore="1475" notAfter="1500">(s. xv ex.)</date>.
+                        This feature is genetic, shared with three closely related books: <bibl
+                           type="MS">Gloucester Cathedral MS 5;</bibl>
+                        <bibl type="MS">Manchester, John Rylands University Library, MS Eng.
+                           1;</bibl> and <bibl type="MS">BodL, MS Digby 230.</bibl>
+                     </note>
+                  </msItem>
+                  <msItem n="2" xml:id="St_Johns_College_MS_6-item2">
+                     <locus from="134ra" to="134ra">Fol. 134<hi rend="superscript">ra</hi></locus>
+                     <incipit>Pees makeþ plente | Plente makyþ pryde</incipit>
+                     <note><bibl>IMEV 2742</bibl>, written in rubricator’s red; <bibl>ed. Bergen,
+                           26</bibl>, from the related <bibl type="MS">Digby 230</bibl>, where it
+                        appears in the same position and format.</note>
+                  </msItem>
+               </msContents>
+               <physDesc>
+                  <p><secFol>hym to releue</secFol></p>
+                  <objectDesc form="codex">
+                     <supportDesc material="perg">
+                        <support>Vellum (FSOS/FHHF).</support>
+                        <extent>Fols. i + <measure type="leaf" quantity="134">134</measure> + i
+                           (numbered fol. ii).<dimensions type="leaf" unit="mm">
+                              <height quantity="390">390</height>
+                              <width quantity="280">280</width>
+                           </dimensions>
```

Comment:
It is not necessary to include the quantity attribute if it is exactly the same as the content of the element (does no harm to include it, but may save you some time not to)